### PR TITLE
MAGECLOUD-4996: Document backwards incompatible changes related to Cloud Docker with ECE-Tools 2002.1.0

### PR DESCRIPTION
**Pull request by michaelcasey316**
## Purpose of this pull request

This pull request (PR) adds backwards incompatible change notes for Magento Cloud Docker 1.0.0.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/release-notes/backward-incompatible-changes.html

whatsnew
Provided information about backward incompatible changes introduced in the Docker-1.0.0 release.

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@michaelcasey316 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/release-notes/backward-incompatible-changes.md [hunk 1]
```diff
@@ -43,3 +43,37 @@ In earlier releases of {{ site.data.var.ct }}, you could use the `m2-ece-build`
...
+
+Workaround: Upgrade to PHP 7.1 or higher on your host machine.
+
+#### Renamed bin/docker
+Change: The command `bin/docker` has been renamed to `bin/magento-docker`.
+
+Workaround: Replace instances of `bin/docker` with `bin/magento-docker` in your scripts and commands.
+
+#### Docker command change
+Change: `bin/magento-docker` commands now remove the containers they create after for one-off commands.
```
[the remark this answers is not in the document]
Comment (michaelcasey316) [reply src/cloud/release-notes/backward-incompatible-changes.md]: I didn't mean to put "for" at all.

**Comment by michaelcasey316** [comment]
I made the changes you requested and fixed a typo.

File: src/cloud/release-notes/backward-incompatible-changes.md [hunk 2]
```diff
@@ -23,23 +38,75 @@ The following table provides information about environment variables and other e
...
--  **New magento/magento-cloud-docker package**–We moved Docker development functionality to a separate package in {{ site.data.var.ct }} v2002.1.0. See [Release notes for magento/magento-cloud-docker]({{ site.baseurl }}/cloud/release-notes/mcd-release-notes.html).
+   -  Remove the `magento-db` volume and all associated volumes when you shut down the containers:
 
-### Patches changes
+      ```bash
+      docker-compose -v
+      ```
 
-The {{site.data.var.mcp}} package bundles all patches available from the [Magento Technical resources](https://magento.com/tech-resources/download) page. If you have downloaded any Magento-supplied patches from this site and copied them into your Magento project, remove them to prevent conflicts.
+-  **Override file synchronization settings for archive and backup files**–Archive and backup files with the following extensions are no longer synchronized when using docker-sync or mutagen:  `*.sql`, `*.gz`, `*.zip`, and `*.bz2`.  You can override the default file synchronization for these file types by renaming the file to end with a different extension, for example `synchronize-me.zip-backup`.
```
Comment (meker12) [inline src/cloud/release-notes/backward-incompatible-changes.md]:
```suggestion
-  **Override file synchronization settings for archive and backup files**–Archive and backup files with the following extensions are no longer synchronized when using docker-sync or mutagen:  `*.sql`, `*.gz`, `*.zip`, and `*.bz2`.  You can override the default file synchronization for these file types by renaming the file with a different extension, for example `synchronize-me.zip-backup`.
```

**Pull request merged**